### ISSUE TITLE
how to get a multi-turn response example

### ISSUE THREAD

**Issue by camphora** [nathanrchn/perplexityai#31]
Truly an amazing project. However the examples in the Readme only shows how to get results with one question and one answer, can anyone show how to get answer from a multi-turn question. 
For instance:  
Q: how to cook an egg?
A: xx
Q: what about fish?
A: <that is what I want>

**Comment by ArtDoctor** [comment]
I don't think it supports multi-turn example yet. You can try to create multiple clients and do it asynchronously, manually.
